### DISCUSSION TITLE
Random Card Mode

### DISCUSSION THREAD

**Discussion by zachleboeuf** [AdriaGual/marvel-snap-bot#51] [Ideas]
You explained to me how to edit the code to make it play random cards without needing images. I think it would be a nice feature to add into the bot natively. I know a lot of people probably want it for booster farming without having to collect all the images.

I also can't get the random card mode to work.

**Comment by eimia-q** [comment]
Hi, I came to ask the same thing, if it was possible to just run on random and not get card or field info. 

My only issue would be the risk of ban for running through lots of games too quickly, which I think is what happened with the auto-retreat version.
